core/lib/spree/testing_support/common_rake.rb

```diff
@@ -17,8 +13,31 @@ def initialize
 
         force_rails_environment_to_test
 
-        Spree::DummyGenerator.start ["--lib_name=#{ENV['LIB_NAME']}", "--quiet"]
-        Solidus::InstallGenerator.start ["--lib_name=#{ENV['LIB_NAME']}", "--auto-accept", "--with-authentication=false", "--payment-method=none", "--migrate=false", "--seed=false", "--sample=false", "--quiet", "--user_class=#{args[:user_class]}"]
+        Spree::DummyGenerator.start [
+          "--lib-name=#{ENV['LIB_NAME']}",
+          "--quiet",
+        ]
+
+        # While the dummy app is generated the current directory
+        # within ruby is changed to that of the dummy app.
+        sh({
+          'SKIP_SOLIDUS_BOLT' => '1',
+          'FRONTEND' => 'solidus_frontend',
```

Comment:
The [master version](https://github.com/nebulab/solidus/blob/29ddabbee282369daf990743dad96d2ce170ac0b/core/lib/spree/testing_support/common_rake.rb#L28) sets `FRONTEND` to `ENV['FRONTEND']` if that is available. Shouldn't we do that here alos? The [SSF 3.2 patch](https://github.com/solidusio/solidus_starter_frontend/blob/09dc12a4d8df600e9002586297f5e1408e964a2e/solidus_3_2_patch.rb#L17) accepts the `FRONTEND` environment variable.

Reply:
Good point! Although it's probably kinda optional as we confirmed that is not actually feasible to build this kind of dummy app with SSF (it breaks due to modifications to the Gemfile and other changes).

So we might even revert that change on master (maybe eventually deprecate the whole spec/dummy approach or externalize it.

Reply:
Okay. Thanks for the explanation @elia !